src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -114,6 +121,28 @@ public virtual Task Init(INethermindApi nethermindApi)
             return Task.CompletedTask;
         }
 
+        private void MigrateSecondsPerSlot()
+        {
+            ulong defaultValue = _blocksConfig.GetDefaultValue<ulong>(nameof(IBlocksConfig.SecondsPerSlot));
+            if (_blocksConfig.SecondsPerSlot != _mergeConfig.SecondsPerSlot)
+            {
+                if (_blocksConfig.SecondsPerSlot == defaultValue)
+                {
+                    _blocksConfig.SecondsPerSlot = _mergeConfig.SecondsPerSlot;
```

Comment:
I assume the default for merge config is the same as blocks config right? As in, if `_mergeConfig.SecondsPerSlot == _mergeConfig.GetDefaultValue<ulong>(nameof(IMergeConfig.SecondsPerSlot));` would it cause a problem?

Reply:
Yes the default for both is the same. otherwise, this code wont work.

Reply:
Wouldn't it be easier to make it null'able? Then null would be default value. And you can treat `null` as 12 after during migration.